setup.cfg

```diff
@@ -11,12 +11,11 @@ long_description = file: README.rst
 [options]
 zip_safe = False
 packages = find:
-python_requires = >=3.6
+python_requires = >=3.9
```

Comment:
I can undo this?

Reply:
I think so, unless we need to, we might as well keep this in line with towncrier / sphinx?

Reply:
TC set to python 3.8